package.json

```diff
@@ -101,7 +101,7 @@
     "proxyquire": "^2.1.3",
     "should": "^13.2.3",
     "svelte": "^3.38.2",
-    "typescript": "^4.0.5"
+    "typescript": "^5.1.6"
```

Comment:
Upgrade to TypeScript 5 is not neccessary and may break some type definitions.
Please use `^4.9.0`.